### PR TITLE
Added "period" for clarity of what a "full stop" is.

### PR DESCRIPTION
**Pull request by sconosciute**
### Description

Tiny change, added the word "period" for clarity of what a "full stop" character is to avoid confusion for NA English readers.

### Motivation

I have only rarely seen a period referred to as a full stop, despite it being the technically correct name for the character. I believe this adds a touch of clarity for readers who may be unfamiliar with the term.

### CODE REVIEW

File: files/en-us/learn/css/first_steps/getting_started/index.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ So far, we have styled elements based on their HTML element names. This works as
    </ul>
    ```
 
-2. In your CSS, you can target the class of `special` by creating a selector that starts with a full stop character. Add the following to your CSS file:
+2. In your CSS, you can target the class of `special` by creating a selector that starts with a period/full stop character. Add the following to your CSS file:
```
Comment (wbamberg) [inline files/en-us/learn/css/first_steps/getting_started/index.md:154]:
```suggestion
2. In your CSS, you can target the class of `special` by creating a selector that starts with a period. Add the following to your CSS file:
```

We don't need "full stop" at all. This was probably written by a British person :).

**Comment by bsmth** [comment]
Thanks a lot! Let's remove "full stop" as Will suggests :)

**Comment by sconosciute** [comment]
Didn't realize I could just click "commit suggestion" 😂. In any case, change is made!

**Review by wbamberg — APPROVED** [review]
👍 thank you @sconosciute !

**Pull request merged**